### ISSUE TITLE
Compile issues 1.3.6 on Centos6

### ISSUE THREAD

**Issue by tsmgeek** [proftpd/proftpd#520]
Trying to compile 1.3.6 on centos with latest mod_vroot but get the following error on make. I have tried different versions of mod_vroot on git but cant get it to work. Is it still supported or built into 1.3.6?

> mod_vroot.c:32:23: error: mod_vroot.h: No such file or directory
> In file included from mod_vroot.c:33:
> ../include/privs.h:61: error: expected ')' before 'const'
> In file included from mod_vroot.c:34:
> alias.h:32: error: expected declaration specifiers or '...' before 'size_t'
> alias.h:33: error: expected declaration specifiers or '...' before 'size_t'
> alias.h:43: error: expected ')' before '*' token
> In file included from mod_vroot.c:35:
> path.h:31: error: expected ')' before '*' token
> path.h:32: error: expected declaration specifiers or '...' before 'size_t'
> path.h:36: error: expected ')' before '*' token
> path.h:40: error: expected ')' before '*' token
> mod_vroot.c: In function 'handle_vrootaliases':
> mod_vroot.c:85: warning: implicit declaration of function 'vroot_path_lookup'
> mod_vroot.c:91: error: 'MOD_VROOT_VERSION' undeclared (first use in this function)
> mod_vroot.c:91: error: (Each undeclared identifier is reported only once
> mod_vroot.c:91: error: for each function it appears in.)
> mod_vroot.c: In function 'set_vrootoptions':
> mod_vroot.c:185: error: 'VROOT_OPT_ALLOW_SYMLINKS' undeclared (first use in this function)
> mod_vroot.c: In function 'vroot_log_retr':
> mod_vroot.c:262: warning: implicit declaration of function 'vroot_path_get_base'
> mod_vroot.c:262: warning: assignment makes pointer from integer without a cast
> mod_vroot.c:267: warning: implicit declaration of function 'vroot_realpath'
> mod_vroot.c:267: warning: assignment makes pointer from integer without a cast
> mod_vroot.c: In function 'vroot_log_stor':
> mod_vroot.c:293: warning: assignment makes pointer from integer without a cast
> mod_vroot.c:298: warning: assignment makes pointer from integer without a cast
> mod_vroot.c: In function 'vroot_pre_pass':
> mod_vroot.c:352: error: 'MOD_VROOT_VERSION' undeclared (first use in this function)
> mod_vroot.c:352: error: expected ')' before string constant
> mod_vroot.c:357: error: expected ')' before string constant
> mod_vroot.c:363: error: 'vroot_fsio_stat' undeclared (first use in this function)
> mod_vroot.c:364: error: 'vroot_fsio_lstat' undeclared (first use in this function)
> mod_vroot.c:365: error: 'vroot_fsio_rename' undeclared (first use in this function)
> mod_vroot.c:366: error: 'vroot_fsio_unlink' undeclared (first use in this function)
> mod_vroot.c:367: error: 'vroot_fsio_open' undeclared (first use in this function)
> mod_vroot.c:371: error: 'vroot_fsio_link' undeclared (first use in this function)
> mod_vroot.c:372: error: 'vroot_fsio_readlink' undeclared (first use in this function)
> mod_vroot.c:373: error: 'vroot_fsio_symlink' undeclared (first use in this function)
> mod_vroot.c:374: error: 'vroot_fsio_truncate' undeclared (first use in this function)
> mod_vroot.c:375: error: 'vroot_fsio_chmod' undeclared (first use in this function)
> mod_vroot.c:376: error: 'vroot_fsio_chown' undeclared (first use in this function)
> mod_vroot.c:378: error: 'vroot_fsio_lchown' undeclared (first use in this function)
> mod_vroot.c:380: error: 'vroot_fsio_chdir' undeclared (first use in this function)
> mod_vroot.c:381: error: 'vroot_fsio_chroot' undeclared (first use in this function)
> mod_vroot.c:382: error: 'vroot_fsio_utimes' undeclared (first use in this function)
> mod_vroot.c:383: error: 'vroot_fsio_opendir' undeclared (first use in this function)
> mod_vroot.c:384: error: 'vroot_fsio_readdir' undeclared (first use in this function)
> mod_vroot.c:385: error: 'vroot_fsio_closedir' undeclared (first use in this function)
> mod_vroot.c:386: error: 'vroot_fsio_mkdir' undeclared (first use in this function)
> mod_vroot.c:387: error: 'vroot_fsio_rmdir' undeclared (first use in this function)
> mod_vroot.c: In function 'vroot_post_pass':
> mod_vroot.c:399: error: 'MOD_VROOT_VERSION' undeclared (first use in this function)
> mod_vroot.c:400: error: expected ')' before string constant
> mod_vroot.c:403: error: expected ')' before string constant
> mod_vroot.c: In function 'vroot_post_pass_err':
> mod_vroot.c:432: error: 'MOD_VROOT_VERSION' undeclared (first use in this function)
> mod_vroot.c:433: error: expected ')' before string constant
> mod_vroot.c:436: error: expected ')' before string constant
> mod_vroot.c: In function 'vroot_exit_ev':
> mod_vroot.c:451: warning: implicit declaration of function 'vroot_fsio_free'
> mod_vroot.c: In function 'vroot_sess_init':
> mod_vroot.c:478: error: 'MOD_VROOT_VERSION' undeclared (first use in this function)
> mod_vroot.c:479: error: expected ')' before string constant
> mod_vroot.c:484: error: expected ')' before string constant
> mod_vroot.c:489: error: expected ')' before string constant
> mod_vroot.c:496: warning: implicit declaration of function 'vroot_alias_init'
> mod_vroot.c:497: warning: implicit declaration of function 'vroot_fsio_init'
> mod_vroot.c: At top level:
> mod_vroot.c:575: error: 'MOD_VROOT_VERSION' undeclared here (not in a function)

**Comment by Castaglia** [comment]
First, please report issues for `mod_vroot` in that module's GitHub repo:

  https://github.com/Castaglia/proftpd-mod_vroot

Second, are you using the latest code for `mod_vroot` from the above repo?

**Comment by Castaglia** [comment]
Closing this in favor of the `mod_vroot` [Issue  #15](https://github.com/Castaglia/proftpd-mod_vroot/issues/15).
